sdk/storage/storage-file-share/src/Clients.ts

```diff
@@ -3896,7 +3914,12 @@ export class ShareFileClient extends StorageClient {
       const res = assertResponse<RawFileDownloadResponse, FileDownloadHeaders>({
         ...rawResponse,
         _response: (rawResponse as any)._response, // _response is made non-enumerable,
-        fileMode: parseOctalFileMode(rawResponse.fileMode),
+        posixProperties: {
+          fileMode: parseOctalFileMode(rawResponse.fileMode),
+          owner: rawResponse.owner,
+          group: rawResponse.group,
+          linkCount: rawResponse.linkCount,
+        },
```

Comment:
Download should have file type?

Reply:
Service doesn't return file type for download. Original download headers definition doesn't include file type either.